server_tests/test/clientTests.js

```diff
@@ -1159,18 +1159,30 @@ describe("RUST API TEST SUITE", function() {
 
         it("Expect success. /osc/commands/_bublPoll returns immediately if no waitTimeout argument is provided", function() {
             this.timeout(timeoutValue);
-            var fingerprint = '';
             var deferred = Q.defer();
+            var commandId = '';
+            var fingerprint = '';
 
             return Q.all([testClient.takePicture(sessionId, function(res) {
-                var commandId = res.body.id;
-                testClient.bublPoll(res.body.id, fingerprint)
-                .then( function onSuccess (res) {
-                    validate.bublPoll(res.body);
-                    assert.notEqual(res.body.fingerprint, fingerprint);
-                    assert.equal(res.body.command.id, commandId);
-                })
-                .then(deferred.resolve, deferred.reject)
+                if (commandId === '') {
+                    commandId = res.body.id;
+                    testClient.bublPoll(commandId, fingerprint)
+                    .then( function onSuccess (res) {
+                        validate.bublPoll(res.body);
```

Comment:
We probably should also validate the command state data here:

    validate.inProgress(res.body.command, schema.names.cameraTakePicture)